### ISSUE TITLE
[v3.0.0-alpha.3] Fatal error: openssl signature could not be verified: openssl public key could not be read

### ISSUE THREAD

**Issue by o-alquimista** [phpDocumentor/phpDocumentor2#2105]
At the root directory of my Symfony 4 project, I run:
```
$ php phpDocumentor.phar -d src/ -t docs/
```
And get a fatal error:
```
Fatal error: Uncaught PharException: phar "C:\wamp64\www\devdungeon\phpDocumentor.phar" openssl signature could not be verified: openssl public key could not be read in C:\wamp64\www\devdungeon\phpDocumentor.phar:10

Stack trace:
#0 C:\wamp64\www\devdungeon\phpDocumentor.phar(10): Phar::mapPhar('box-auto-genera...')
#1 {main}
  thrown in C:\wamp64\www\devdungeon\phpDocumentor.phar on line 10
```

This is my project: https://github.com/o-alquimista/devdungeon
PHP 7.3.5
Windows 7 64-bit
phpDocumentor v3.0.0-alpha.3 in a PHAR file.

**Comment by jaapio** [comment]
You need to place the key which is in the release as well in the same directory as you installed the phar.

**Comment by o-alquimista** [comment]
It works now. Sorry.

One question, if I may.
There are no compilation errors when using this version, while on v2 it always complained about missing summaries in some docblocks. Is this the new normal in v3?

**Comment by jaapio** [comment]
Yes we removed the validation. It will come back at some point but not in the 3.0 release. The validation did not fit in the massive amount of internal changes we had to do to support all kind of new language features. 

There are many other great tools that can help you with the validation of your code style. At this point we think we can better be good at building a compleet documentation than focus on validation.
